apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+require('babel-polyfill');
+window.React = require('react');
+window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```

Comment:
I was able to do `import React from 'react';` just fine, but removing the line at https://github.com/code-dot-org/code-dot-org/pull/10567/files#diff-26cdc4306e5d33ea4df1299321d8852aR443 from gruntfile.js gives this console error when attempting to load the page: `Uncaught ReferenceError: React is not defined`.